### PR TITLE
[IIachimov] Added Generics Task

### PR DESCRIPTION
**Pull request by IIachimov**
Please review the current pull request for Generics task

### CODE REVIEW

File: iulianIachimov/src/main/java/genericstask/Main.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import genericstask.model.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+
+  public static void main(String[] args) {
+// first task
+    Integer[] arrayOfInteger = new Integer[]{1, 2, 3, 48, 89, 4563};
```
Comment (solonarieugeniu) [inline iulianIachimov/src/main/java/genericstask/Main.java]: no need in `new`

File: iulianIachimov/src/main/java/genericstask/Main.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    String[] arrayOfString = new String[]{"Table", "Chair", "Cupboard"};
```
Comment (solonarieugeniu) [inline iulianIachimov/src/main/java/genericstask/Main.java]: same

File: iulianIachimov/src/main/java/genericstask/Main.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+    Declaration.printArray(arrayOfInteger);
```
Comment (solonarieugeniu) [inline iulianIachimov/src/main/java/genericstask/Main.java]: static import please

File: iulianIachimov/src/main/java/genericstask/model/Declaration.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package genericstask.model;
+
+public class Declaration {
```
Comment (solonarieugeniu) [inline iulianIachimov/src/main/java/genericstask/model/Declaration.java]: please rename, e.g CustomArrayUtils

File: iulianIachimov/src/main/java/genericstask/Main.java (hunk 1 above)
Comment (IIachimov) [reply iulianIachimov/src/main/java/genericstask/Main.java]: Fixed

File: iulianIachimov/src/main/java/genericstask/Main.java (hunk 2 above)
Comment (IIachimov) [reply iulianIachimov/src/main/java/genericstask/Main.java]: Fixed

File: iulianIachimov/src/main/java/genericstask/Main.java (hunk 3 above)
Comment (IIachimov) [reply iulianIachimov/src/main/java/genericstask/Main.java]: This is done

File: iulianIachimov/src/main/java/genericstask/model/Declaration.java (hunk 4 above)
Comment (IIachimov) [reply iulianIachimov/src/main/java/genericstask/model/Declaration.java]: Renamed according to suggestion

**Review by abadarau — APPROVED** [review]

**Review by dvmunteanu — APPROVED** [review]

**Review by gaenache — APPROVED** [review]

**Review by constantin73 — APPROVED** [review]

**Review by asecu — APPROVED** [review]

**Pull request merged**